src/main/app/forms/models/contactDetails.ts

```diff
@@ -12,10 +11,11 @@ export class ValidationErrors {
 export class ContactDetails implements Serializable<ContactDetails> {
 
   @IsOptional()
-  @IsPhone({ message: ValidationErrors.PHONE_NUMBER_NOT_VALID })
+  @IsPhoneNumber({ message: ValidationErrors.PHONE_NUMBER_NOT_VALID })
   phoneNumber?: string
 
   @IsOptional()
+  @IsNotEmpty()
```

Comment:
I bet we can remove it. Empty value is deserialised to undefined which is permitted. Value with single whitespaces only is not forbidden by email validator.

Reply:
There is a unit test that checks that an empty email address is rejected (Line 80 here -> https://github.com/hmcts/cmc-legal-rep-frontend/blob/master/src/test/app/forms/models/contactDetails.ts). 

Should this be left as is, or have the unit test removed if it is irrellevant?

Reply:
That tests did not reflect real use so I removed it. The way deserialisation is implemented makes empty email addresses to be resolved to undefined and the validation rules should permit such case.